src/Symfony/Component/Asset/Context/NullContext.php

```diff
@@ -18,6 +18,13 @@
  */
 class NullContext implements ContextInterface
 {
+    public function __construct()
+    {
+        if (__CLASS__ !== get_class($this)) {
```

Comment:
Maybe to be safe you could add a `@final` tag and give a thumb up for https://youtrack.jetbrains.com/issue/WI-31225 :P